package/yast2-packager.changes

```diff
@@ -1,4 +1,11 @@
 -------------------------------------------------------------------
+Thu Aug 18 10:55:06 CEST 2016 - schubi@suse.de
+
+- Do not raise an exception if no license is available (no licence
+  dir at all) to accept. This is still a fix for bnc#993285.
```

Comment:
a new bug has been filed for this,  https://bugzilla.suse.com/show_bug.cgi?id=994309